srcpkgs/a4/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'a4'
+pkgname=a4
+version=0.2.1
+revision=1
+#archs="i686 x86_64"
```

Comment:
Remove the commented lines.

Reply:
I apologize, I'm new to this tasks! 

Commended lines removed!

Many thanks

Max

Reply:
I think to have cleaned correctly the template, can you please confirm_ 

many thanks 
Max